main.go

```diff
@@ -7,13 +7,18 @@ import (
 
 	"github.com/manifoldco/torus-cli/cmd"
 	"github.com/manifoldco/torus-cli/config"
+	"github.com/manifoldco/torus-cli/prefs"
+	"github.com/manifoldco/torus-cli/ui"
 )
 
 func main() {
 	cli.VersionPrinter = func(ctx *cli.Context) {
 		cmd.VersionLookup(ctx)
 	}
 
+	preferences, _ := prefs.NewPreferences(true)
```

Comment:
yeah this was the thing I was talking to you about, having the default booleans appear in the rc file after save.